httprouter/bearerstdapi/bearerstdapi.go

```diff
@@ -66,6 +104,64 @@ func (b *BearerStandardAPI) ProcessData(req *http.Request) (interface{}, error)
 	}, nil
 }
 
+// RegisterMethod adds a new method under the URL pattern.
+// The pattern can contain variable names (starting with #), such as /send/#name/hello
+// The pattern can also contain wildcard at the end of the path, such as /send/#name/*
+// The accessType can be of type private, public or admin.
+func (b *BearerStandardAPI) RegisterMethod(pattern, HTTPmethod string, accessType string, handler BearerStdAPIhandler) error {
+	if pattern[0] != '/' {
+		return fmt.Errorf("pattern must start with /")
```

Comment:
in my opinion these errors can only happen with mistakes like typos in constant strings, and RegisterMethod is only called when the service starts - so IMO they should be panics rather than errors. Either way, the application will fail to start. The bonus of panics is that they clarify "you called me with invalid input", and we don't have to do as much error checking that simply bubbles up to os.Exit.

Reply:
makes sense